osprey_flask_app/templates/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
```

Comment:
Is there a way to encapsulate just the progress bar part of this template so that it could be included in other apps (like a Jupyter notebook)?